extensions/graph/graph.ui.runtime.tsx

```diff
@@ -0,0 +1,45 @@
+import { ComponentType } from 'react';
+import { UIRuntime } from '@teambit/ui';
+import { Slot, SlotRegistry } from '@teambit/harmony';
+
+import { ComponentAspect, ComponentUI, ComponentModel } from '@teambit/component';
+import { GraphAspect } from './graph.aspect';
+import { GraphSection } from './ui/graph.section';
+
+export interface ComponentWidgetProps extends React.HTMLAttributes<HTMLDivElement> {
+  component: ComponentModel;
+}
+export type ComponentWidget = ComponentType<ComponentWidgetProps>;
+export type ComponentWidgetSlot = SlotRegistry<ComponentWidget>;
+
+/**
+ * Presents dependencies graph in the component page
+ */
+export class GraphUI {
+  /**
+   * adds plugins to component nodes
+   * @param value
+   */
+  registerComponentWidget(value: ComponentWidget) {
+    this.componentWidgetSlot.register(value);
+  }
+
+  constructor(private componentWidgetSlot: ComponentWidgetSlot) {}
+  static dependencies = [ComponentAspect];
+  static runtime = UIRuntime;
+  static slots = [Slot.withType<ComponentWidget>()];
+  static async provider([componentUI]: [ComponentUI], config, [componentWidgetSlot]: [ComponentWidgetSlot]) {
+    const graphUI = new GraphUI(componentWidgetSlot);
+
+    // TODO - remove feature flag when data provider works for all workspaces
+    if (window.location.pathname !== '/enable-graph') return;
```

Comment:
why pathname and not query param?

Reply:
it's easier to got to localhost:3000/enable-graph as a landing page